### ISSUE TITLE
NTR: 2-trans,4-cis-decadienoylcarnitine present in urine (Lab: chemistry: carnitine compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8567]
New term request 2-trans,4-cis-decadienoylcarnitine present in urine (segal_050103220950):

**Comment by pnrobinson** [comment]
@MickeySegal
is this a current clinical assay?
Here (https://omim.org/entry/616034) I see one mention from a research paper from 1990 but cannot find subsequent mentions

**Comment by MickeySegal** [comment]
Acylcarnitine testing by mass spectroscopy is a current clinical assay: https://www.labcorp.com/tests/070228/acylcarnitine-profile-quantitative-plasma

It sounds like the metabolite was picked up in this clinical test.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000477
- label: Elevated urine 2-trans,4-cis-decadienoylcarnitine level
- parents: [Abnormal concentration of acylcarnitine in the urine (HP:0500170)](https://hpo.jax.org/app/browse/term/HP:0500170)
- pmid: 
- def: The amount of 2-trans,4-cis-decadienoylcarnitine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8567
- synonyms:
